doc/specs/#597 - Tab Sizing.md

```diff
@@ -0,0 +1,95 @@
+---
+author: Kayla Cinnamon @cinnamon-msft
+created on: 2020-01-03
+last updated: 2020-01-03
+issue id: 597
+---
+
+# Tab Sizing
+
+## Abstract
+
+This spec outlines the tab sizing feature. This is an application-level feature that is not profile-specific (at least for now).
+
+Global properties that encompass tab sizing:
+
+* `tabWidthMode` (accepts pre-defined values for tab sizing behavior)
+* `tabWidthMin` (can never be smaller than the icon width)
+* `tabWidthMax` (can never be wider than the tab bar)
+
+Acceptable values for `tabWidthMode`:
+
+* [default] `even` (all tabs are sized the same, regardless of tab title length)
+* `titleLength` (width of tab contains entire tab title)
+
+## Inspiration
+
+Other browsers and terminals have varying tab width behavior, so we should give people options.
+
+## Solution Design
+
+`tabWidthMode` will be a global setting that will accept the following strings:
+
+* `even`

```

Comment:
Why not use `equal` here?  That's the TabView setting so presumably, they put some thought behind that name.  For some devs, `even` will make them think "evenly divisible by 2" and implies there could be an `odd` setting.